tests/portability/cmsis_rtos_v1/src/timer.c

```diff
@@ -92,7 +92,7 @@ void test_timer(void)
 	/* The first firing of the timer should be ignored.
```

Comment:
Comment is now void

Reply:
Nice catch, I've removed the comments in the latest commit.